kieker-monitoring/test/kieker/test/monitoring/junit/writer/serializer/BinarySerializerTest.java

```diff
@@ -48,7 +48,7 @@ public BinarySerializerTest() {
 		// Default Constructor
 	}
 
-	@Test
+	//@Test
```

Comment:
Do we need a ticket for revisiting these deactivated tests or can they be removed already?

Reply:
No. We have a cleanup process for this in place. The test will go when we drop the old serializer classes are removed. They are marked as deprecated.